### ISSUE TITLE
Inline math in the spec

### ISSUE THREAD

**Issue by davidcarlisle** [w3c/mathml#362]
At various points in the document we have small inline expressions. In MathML3 we used the document italic (`<i class="var">`)  unless there were any fractions or Greek in which case we used vertically centred images.

<div class="notinspec">

Alternatives seem to be:

* Use inline images
* Use document text sans serif italic (sloped)  
* Use document serif Italic ie css style `.var` to be (any?) serif font
* Use Unicode Math Italic Block  (possibly using the reader's default font for this range)
* Use MathML Core

The current draft is using the second of these ( `<i` giving sans serif italic). I think that's fine for casual expression references in a sans serif document, but opening the issue so we can decide and treat them all the same way.

```
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">e<
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">e<
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">e<
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">e<
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">e<
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">e<
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">e<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">f<
src/contm-new.html:<i class="var">f<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">vector(complex)<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">cdbase<
src/contm-new.html:<i class="var">cd-name<
src/contm-new.html:<i class="var">symbol-name<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">z<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">F<
src/contm-new.html:<i class="var">G<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">F<
src/contm-new.html:<i class="var">G<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">z<
src/contm-new.html:<i class="var">z<
src/contm-new.html:<i class="var">z<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">z<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">URI<
src/contm-new.html:<i class="var">f(f(f(a,a),f(a,a)),f(f(a,a),f(a,a)))<
src/contm-new.html:<i class="var">&#x03bb;<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">f<
src/contm-new.html:<i class="var">&#x03bb;<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">g<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">g<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">g<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">&#x03bb;<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">f<
src/contm-new.html:<i class="var">&#x03bb;<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">g<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">g<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">g<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">C<
src/contm-new.html:<i class="var">f<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">f<
src/contm-new.html:<i class="var">C<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">c<
src/contm-new.html:<i class="var">d<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">c<
src/contm-new.html:<i class="var">d<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">q<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">q<
src/contm-new.html:<i class="var">r<
src/contm-new.html:<i class="var">r<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">r<
src/contm-new.html:<i class="var">q<
src/contm-new.html:<i class="var">r<
src/contm-new.html:<i>x<
src/contm-new.html:<i>x<
src/contm-new.html:<i>y<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">r<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">q<
src/contm-new.html:<i class="var">r<
src/contm-new.html:<i class="var">r<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">r<
src/contm-new.html:<i class="var">b<
src/contm-new.html:<i class="var">a<
src/contm-new.html:<i class="var">n<
src/contm-new.html:<i class="var">n<
src/contm-new.html:<i class="var">n<
src/contm-new.html:<i class="var">n<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">y<
src/contm-new.html:<i>x<
src/contm-new.html:<i>x<
src/contm-new.html:<i>y<
src/contm-new.html:<i class="var">i<
src/contm-new.html:<i>i<
src/contm-new.html:<i>&#x03c0;<
src/contm-new.html:<i>&#x03b3;<
src/contm-new.html:<i class="var">n<
src/contm-new.html:<i class="var">n<
src/contm-new.html:<i class="var">n<
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">x<
src/contm-new.html:<i class="var">H<
src/contm-new.html:<i class="var">x<
src/mixing.html:<i class="var">a<
src/mixing.html:<i class="var">b<
src/mixing.html:<i class="var">c<
src/mixing.html:<i class="var">d<
src/mixing.html:<i class="var">a<
src/mixing.html:<i class="var">b<
src/mixing.html:<i class="var">c<
src/mixing.html:<i class="var">d<
src/operator-dict.html:<i class="var">x<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">a<
src/presentation-markup.html:<i class="var">b<
src/presentation-markup.html:<i class="var">f<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">f<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">f<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">f<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">a<
src/presentation-markup.html:<i class="var">b<
src/presentation-markup.html:<i class="var">f<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">y<
src/presentation-markup.html:<i class="var">f<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">y<
src/presentation-markup.html:<i class="var">m<
src/presentation-markup.html:<i class="var">a<
src/presentation-markup.html:<i class="var">b<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">y<
src/presentation-markup.html:<i class="var">z<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">y<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">y<
src/presentation-markup.html:<i class="var">a<
src/presentation-markup.html:<i class="var">b<
src/presentation-markup.html:<i class="var">x<
src/presentation-markup.html:<i class="var">y<
src/presentation-markup.html:<i class="var">k<
src/presentation-markup.html:<i class="var">l<
src/presentation-markup.html:<i class="var">n<
src/presentation-markup.html:<i class="var">n<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">n<
src/presentation-markup.html:<i class="var">n<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">n<
src/presentation-markup.html:<i class="var">n<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/presentation-markup.html:<i class="var">i<
src/transform2strict.html:<i class="var">x<
src/transform2strict.html:<i class="var">x<
src/transform2strict.html:<i class="var">x<
src/transform2strict.html:<i class="var">x<
src/transform2strict.html:<i class="var">x<
src/transform2strict.html:<i class="var">f<
src/transform2strict.html:<i class="var">C<
src/transform2strict.html:<i class="var">x<
src/transform2strict.html:<i class="var">&#x3c0;<
src/transform2strict.html:<i class="var">C<
src/transform2strict.html:<i class="var">x<
src/transform2strict.html:<i class="var">x<
src/transform2strict.html:<i class="var">y<
src/world-interactions.html:<i class="var">a<
```

</div>

**Comment by NSoiffer** [comment]
Ideally, they should all be core -- that's would people would do in LaTeX (probably because it's the easiest way to get it done ;-). For anything that is more than a single char, it should definitely be math as that affects the braille. I don't think (but am not sure) that for a single char, the braille is different for literary vs math braille. I could be wrong for some non-ASCII chars though because UEB and Nemeth have some difference there.

Converting most of them to core would just be a global regular expr replace, right? So not too hard to do. Or maybe something that respec can do?

**Comment by davidcarlisle** [comment]
Not impossibly difficult but can't easily be fully automated as you'd need to gather surrounding text eg  cases like these would want `<math>` around the whole expression not just each `<i>`

```
L(<i class="var">i</i>)
```

```
(<i class="var">x</i>+<i class="var">y</i>)<sup>2</sup> 
```

But I'm not sure we should do this. We have added mathml rendering but only for examples, this would be the first use of required mathml support to read the normative text, which might be an unnecessary bar to pass, since most of them are just an italic _x_

**Comment by davidcarlisle** [comment]
closing as fixed by #374  the basic respec support is in place and most of the larger inline math expressions now mathml

**Comment by NSoiffer** [comment]
Anything with a <sup> or <sub> needs to be math as they would not read
properly with a screen reader. Nor would they braille properly.
